imagersite/imagersite/tests.py

```diff
@@ -0,0 +1,148 @@
+import unittest
+from django.test import Client
+from imager_images.models import Photo
+from imager_profile.models import User
+import factory
+from faker import Factory as FakeFaker
+from django.conf import settings
+import os
+from django.core.mail import outbox
+import re
+
+fake = FakeFaker.create()
+
+
+class PhotoFactory(factory.Factory):
+    class Meta:
+        model = Photo
+    title = fake.sentence()
+    description = fake.paragraph()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = factory.LazyAttribute(lambda a: '{}.{}@example.com'.format(
+                                  a.first_name, a.last_name).lower())
+    username = factory.LazyAttribute(lambda a: '{}{}{}{}'.format(
+                                     a.last_name[:1], a.first_name[1:],
+                                     a.first_name[:1], a.last_name[1:]).lower())
+
+
+class HomeExists(unittest.TestCase):
```

Comment:
You really really really want to import `TestCase` from `django.test`.  The Django test case class has a number of special features that make it work better with databases.  It's quite important to do this.